test/views/pages/gains/PolicyEventPageViewSpec.scala

```diff
@@ -34,8 +34,8 @@ class PolicyEventPageViewSpec extends ViewUnitTest {
 
 
   object Selectors {
+    val hintText = "#main-content > div > div > form > div > fieldset > .govuk-hint"
```

Comment:
we removed .govuk-hint in app/views/pages/gains/PolicyEventPageView.scala.html and added it here. Can we cross check this please